### PR TITLE
Description Summary

### PR DESCRIPTION
**Pull request by SticksPluto**
#Added
DuplicateStrategy in build.gradle (processResources) for the mcmod.info
A mcmod.info File
A pack.mcmeta (for en_us.lang instead of en_US.lang) File

#Fixed
Mod ID Being all CAPS
Changed assets.mwc to
assets, mwc for separate folders instead of a glitch same with textures.gui

### CODE REVIEW

File: src/main/resources/pack.mcmeta [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "pack": {
+        "description": "Heyo watchu looking in the code for anyways look below at _comment for some useless shit :)",
+        "pack_format": 3,
+        "_comment": "A pack_format of 3 should be used starting with Minecraft 1.11. All resources, including language files, should be lowercase (eg: en_us.lang). A pack_format of 2 will load your mod resources with LegacyV2Adapter, which requires language files to have uppercase letters (eg: en_US.lang)."
```
Comment (JustDesoroxxx) [inline src/main/resources/pack.mcmeta:5]: Comment is not needed nor useful and should be entirely removed

File: src/main/resources/mcmod.info [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "modid": "mwc",
+    "name": "Modern Warfare Cubed",
+    "description": "Modern Warfare Cubed, VMW: Forked. PMW",
+    "version": "${version}",
+    "mcversion": "1.12.2",
+    "url": "",
```
Comment (JustDesoroxxx) [inline src/main/resources/mcmod.info:8]: Add the URL of this repo main page

File: src/main/resources/mcmod.info [hunk 3]
```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "modid": "mwc",
+    "name": "Modern Warfare Cubed",
+    "description": "Modern Warfare Cubed, VMW: Forked. PMW",
```
Comment (JustDesoroxxx) [inline src/main/resources/mcmod.info:5]: Please use the real description of this project:

> A well optimized, open source Modern Warfare rewrite, made in light of Vic's Modern Warfare!

File: build.gradle [hunk 4]
```diff
@@ -29,6 +29,7 @@ minecraft {
 
 // Replace version in `mcmod.info`
 processResources {
+    duplicatesStrategy = 'include'
```
Comment (JustDesoroxxx) [inline build.gradle:32]: Please add a space between the `duplicateStrategy` line and the `from()` line

File: src/main/resources/mcmod.info [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    "version": "${version}",
+    "mcversion": "1.12.2",
+    "url": "",
+    "updateUrl": "",
+    "authorList": ["Panda"],
```
Comment (JustDesoroxxx) [inline src/main/resources/mcmod.info:10]: Unsure but maybe @Paneedah would like to be referred as Paneedah, wait for his input on this though

**Review by JustDesoroxxx — CHANGES_REQUESTED** [review]
Just need some correction and it can be merged

File: src/main/resources/mcmod.info (hunk 5 above)
Comment (Paneedah) [reply src/main/resources/mcmod.info:10]: I'd prefer to be referred to as Paneedah, yes.

File: src/main/resources/pack.mcmeta [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+    "pack": {
+        "description": "Heyo watchu looking in the code for anyways look below at _comment for some useless shit :)",
```
[the remark this answers is not in the document]
Comment (SticksPluto) [reply src/main/resources/pack.mcmeta:3]: they can't see the pack.mcmeta ingame so It's a little joke you can go on but I'll edit rn

File: src/main/resources/mcmod.info (hunk 5 above)
[the remark this answers is not in the document]
Comment (SticksPluto) [reply src/main/resources/mcmod.info:10]: got it

File: src/main/resources/mcmod.info [hunk 7]
```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "modid": "mwc",
+    "name": "Modern Warfare Cubed",
+    "description": "Modern Warfare Cubed, VMW: Forked. PMW",
+    "version": "${version}",
+    "mcversion": "1.12.2",
+    "url": "",
+    "updateUrl": "",
```
[the remark this answers is not in the document]
Comment (SticksPluto) [reply src/main/resources/mcmod.info:9]: Got it Removing MCVersion

File: src/main/resources/mcmod.info (hunk 2 above)
Comment (SticksPluto) [reply src/main/resources/mcmod.info:8]: got it

File: src/main/resources/mcmod.info (hunk 3 above)
Comment (SticksPluto) [reply src/main/resources/mcmod.info:5]: done

File: build.gradle (hunk 4 above)
Comment (SticksPluto) [reply build.gradle:32]: got it clean

File: src/main/resources/mcmod.info (hunk 5 above)
Comment (SticksPluto) [reply src/main/resources/mcmod.info:10]: Got it Paneedah

**Review by JustDesoroxxx — APPROVED** [review]

**Pull request merged**